crates/swc_ecma_lints/tests/pass/no_console/1/input.js

```diff
@@ -0,0 +1,8 @@
+object.console.log("message");
+
+(console) => {
+    console.log("message");
+};
+
```

Comment:
Seems like test file is missing output.

Reply:
@kdy1 quite don't understand what you're mean

Reply:
I mean it does not have any reference file to compare with.
`UPDATE=1 cargo test` should generate it, if it's implemented correctly.

Reply:
@kdy1 now results compare is not implemented for lints crate. If I understand correctly I should read output from handler and write it to file with `NormalizedOutput` to compare outputs in future. Now I can't understand how to get output from Handler

Reply:
I think you can use integration test suite of the `swc` crate in this case.

You can add tests in https://github.com/swc-project/swc/tree/74b0624c0c572d462ca7462a3325c6b0f89ae3b5/crates/swc/tests/errors/lints with `.swcrc` which is json file like

```json
{
    "jsc": {
        "lints": {
            "noConsole": "error"
        }
    }
}
```


Note that file name should be `input.js`, and .`swcrc` is searched upward starting from the directory of `input.js`.

After doing so, you can run `UPDATE=1 cargo test` on `swc` crate directory to create test reference files.